sechub-doc/src/docs/asciidoc/sechub-getting-started.adoc

```diff
@@ -0,0 +1,223 @@
+// SPDX-License-Identifier: MIT
+
+include::documents/gen/server-version.adoc[]
+
+include::documents/config.adoc[]
+
+= image:sechub-logo.png[sechub] SecHub Getting Started Guide
+
+include::documents/shared/about_sechub.adoc[]
+
+include::documents/shared/about_documentation_all.adoc[]
+
+//--
+
+== Introduction
+This tutorial is going to teach you how to locally run the SecHub server and the PDS (Product Delegation Server) server for GoSec. + 
+After that, you are going to learn how to scan apps written in 'go' for vulnerabilities. +
+And optionally, install and work with the SecHub extension for VSCodium.
+
+plantuml::diagrams/diagram_behind_the_scenes_sechub.plantuml[format=svg, title=Setup Overview] +
+This Diagram roughly explains how Sechub functions, behind the scenes.
+[WARNING]
+--
+This tutorial is tested and intended for Debian/Ubuntu! +
+Although it should also work on different Linux distributions.
+--
+
+=== Prerequisites
+
+You are going to need: 
+
+* docker for https://docs.docker.com/engine/install/debian/[Debian] / https://docs.docker.com/engine/install/ubuntu/[Ubuntu]
+* docker compose
```

Comment:
Is Docker compose already included in the new Docker?

Reply:
![image](https://user-images.githubusercontent.com/29148934/220914946-18de4485-f073-40a1-9fa4-c750e4a730e1.png)
[Source](https://docs.docker.com/compose/compose-v2/)